KsApi/Service.swift

```diff
@@ -53,7 +53,7 @@ public struct Service: ServiceType {
     GraphQL.shared.configure(
       with: serverConfig.graphQLEndpointUrl,
       headers: self.defaultHeaders,
-      additionalHeaders: { perimeterXClient.headers() }
+      additionalHeaders: self.perimeterXClient.headers()
```

Comment:
What's the reason this change had to be made? Seems like it's doing the same thing but still curious.